Src/DependencyCollector/Shared/Implementation/ProfilerSqlConnectionProcessing.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System;
+    using System.Data.SqlClient;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the calls backs
+    /// received from Profiler instrumentation for SQL Connection.    
+    /// </summary>
+    internal sealed class ProfilerSqlConnectionProcessing : ProfilerSqlProcessingBase
+    {
+        /// <summary>
+        /// Constant command text to return.
+        /// </summary> 
+        private const string SqlConnectionCommandText = "Open";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProfilerSqlConnectionProcessing"/> class.
+        /// </summary>
+        internal ProfilerSqlConnectionProcessing(TelemetryConfiguration configuration, string agentVersion, ObjectInstanceBasedOperationHolder telemetryTupleHolder)
+            : base(configuration, agentVersion, telemetryTupleHolder)
+        {
+        }              
+
+        /// <summary>
+        /// Gets SQL connection resource name.
+        /// </summary>
+        /// <param name="thisObj">The SQL connection.</param>
```

Comment:
i'll leave as is - for me it's more meaningful as i understand that it's exactly 'this' object that i got from the instrumentation

---
In reply to: [98077223](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98077223) [](ancestors = 98077223)